knora/dsplib/models/user.py

```diff
@@ -475,18 +476,17 @@ def fromJsonObj(cls, con: Connection, json_obj: Any):
         in_groups: set[str] = set()
         if json_obj.get('permissions') is not None and json_obj['permissions'].get('groupsPerProject') is not None:
             sysadmin = False
-            project_groups = json_obj['permissions']['groupsPerProject']
-            for project in project_groups:
-                if project == Project.SYSTEM_PROJECT:
-                    if Group.PROJECT_SYSTEMADMIN_GROUP in project_groups[project]:
+            for project_iri, group_memberships in json_obj['permissions']['groupsPerProject'].items():
+                if project_iri == Project.SYSTEM_PROJECT:
+                    if Group.PROJECT_SYSTEMADMIN_GROUP in group_memberships:
                         sysadmin = True
```

Comment:
this could be simplified even further (I think): `sysadmin = (Group.PROJECT_SYSTEMADMIN_GROUP in group_memberships)`